src/env_node.mli

```diff
@@ -19,6 +19,12 @@ val external_ : t -> profile:string -> default:Env.t -> Env.t
 
 val ocaml_flags : t -> profile:string -> expander:Expander.t -> Ocaml_flags.t
 
+val c_flags : t -> profile:string -> expander:Expander.t
+  -> (unit, string list) Build.t
```

Comment:
Nit: if it's a multiline signature, our style is to put an argument on each line. There's many examples in the codebase of this.

Reply:
fixed.